sklearn/decomposition/kernel_pca.py

```diff
@@ -77,28 +74,40 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         When n_components is None, this parameter is ignored and components
         with zero eigenvalues are removed regardless.
 
-    random_state : int seed, RandomState instance, or None, default : None
+    random_state : int seed, RandomState instance, or None, default=None
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
-    n_jobs : int, optional (default = 1)
+    n_jobs : int, default=1
         The number of parallel jobs to run.
-        If ``-1``, then the number of jobs is set to the number of CPU cores.
+        If `-1`, then the number of jobs is set to the number of CPU cores.
```

Comment:
Try rebuilding the docs to double-check. IIRC, double backticks render as monospace fonts, while single backticks render as italics.